Automattic/jetpack
# Social: Pass down description for Social previews

This change is connected to https://github.com/Automattic/wp-calypso/pull/87532

## Proposed changes:

* Passed down description so the preview components can use them

### Other information:


## Jetpack product discussion


https://github.com/orgs/Automattic/projects/733/views/2?pane=issue&itemId=52938448
## Does this pull request change what data or activity we track or use?


No.
## Testing instructions:






* Check out this PR on Jetpack, ~~and for Calypso, check out https://github.com/Automattic/wp-calypso/pull/87532~~
* ~~Follow the instructions here p1680773379311239/1680727892.809789-slack-C04UC7YGPNY~~
* Install the packages with jetpack CLI
* If done, you should see the previews in your wp-admin for Social Notes.
* Check each preview and make sure it looks as expected.
* Create a new normal post, and make sure everything works there too as intended
